browser/src/control/Control.Zotero.js

```diff
@@ -1036,6 +1036,32 @@ L.Control.Zotero = L.Control.extend({
 			this.map.uiManager.showSnackbar(_('Updating citations'));
 	},
 
+	handleUnlinkCitations: function() {
+		var parametes = {};
+		var command = '';
+		if (this.getFieldType() === 'Field') {
+			parametes = {
+				'FieldType': {
+					'type': 'string',
+					'value': 'vnd.oasis.opendocument.field.UNHANDLED'
+				},
+				'FieldCommandPrefix': {
+					'type': 'string',
+					'value': 'ADDIN ZOTERO_'
+				},
+			};
+			command = '.uno:DeleteTextFormFields';
+		}
+		this.map.sendUnoCommand(command, parametes);
```

Comment:
@lpranam what if it is a different type?

Reply:
Core APIs are still in development for others